conference/schedule/models.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, print_function, unicode_literals
+
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from conference.cfp.models import Submission
+
+
+class TalkManager(models.Manager):
+    def get_queryset(self):
+        return super(TalkManager, self).get_queryset()\
+            .select_related(
```

Comment:
Doesn't ``select_related('talk')`` suit? You just need to declare the linked model